corehq/apps/hqwebapp/static/hqwebapp/js/components/inline_edit.js

```diff
@@ -7,6 +7,7 @@
  *  - value: Text to display and edit
  *  - name: HTML name of input
  *  - id: HTML id of input
+ *  - allow_edit: Whether the edit mode should be allowed or not
```

Comment:
Given that this widget's purpose is to edit, I'd expect disabling it to be an unusual case, so for this to be an optional param and to be the inverse, something like `disable` or `disallow_edit`.

Reply:
Updated.